core/tests/webdriverio_utils/waitFor.js

```diff
@@ -143,6 +143,25 @@ var elementAttributeToBe = async function(
   });
 };
 
+var elementStoppedMoving = async function(
+    element, errorMessage
+) {
+  await browser.waitUntil(async function() {
+    var firstValue = await element.getLocation('x');
+    // We need to pause the browser before getting the next
+    // location of element to make sure the elements provide equal
+    // location points only when they stopped moving.
+    // eslint-disable-next-line oppia/e2e-practices
+    await browser.pause(1000);
+    var secondValue = await element.getLocation('x');
+    return await firstValue === secondValue;
```

Comment:
To be true with you, I am not able to understand why you are checking the equality of first value with second value because it seems that it will the result the same x value(in terms of coordinate system) no matter whether it's moving or not(because suppose even it's moving then then also it will moving along the same x axis). May be the flake got resolved because of browser.pause(1000)?

Reply:
If an element on X axis then its x-coordinate will change with respect to time not its Y coordinate, it remains same. That's why I am using X coordinates for comparison instead of Y coordinate as it remains same.

Reply:
I misinterpreted that the element was moving along the y axis.

Can we not have the name of the function to something like waitForRightTransistionToComplete? As waitForElementToBeStopped sounds like it covers all the cases(x, y, etc) which it isn't in this scenario?

Reply:
Done.